app/controllers/articles_controller.rb

```diff
@@ -35,9 +38,14 @@ def destroy
   
   #Crops the text to 160 chars and adds '...' to the end
   def preview_body(body)
-    strip_tags(body)[0..160].gsub(/\s\w+\s*$/,'...')
+    strip_tags(body)[0..155].gsub(/\s\w+\s*$/,'...')
   end
   helper_method :preview_body
+  
+  def check_parent(article)
+    article.user_id == session[:user_id]
+  end
+  helper_method :check_parent
```

Comment:
There's no hard and fast rule about what logic goes where, but I would probably throw this in a method on the Article class (eg: `def parent_is?(user)` or `def authored_by?(user)`). Motivation for that would be the general goal of keeping my controllers as narrowly focused and simple as possible, putting odd logic down into the models where they can be unit tested, and then into other classes in `lib` if the models start to fill up too much.

That said, the functionality here is quite simple, so keep it wherever you think is best, and do some reflection now and then to see how you feel about where complexity is growing in your app.